assets/agenda/actions.ts

```diff
@@ -399,6 +399,18 @@ export function fetchItem(id: any) {
     };
 }
 
+export function fetchItemsById(ids: Array<string>): Promise<IRestApiResponse<IAgendaItem>> {
+    return server.get(`/agenda/search?ids=${ids.join(', ')}`);
+}
+
+export function fetchItemsByIdToRedux(ids: string[]): (dispatch: any) => Promise<void> {
```

Comment:
The 1st function is using `Array<string>` while the 2nd function is using `string[]`, though they're exactly the same. I believe by convention we're using `Array<type>` syntax. It would be good to stick to the one syntax as opposed to mixing different styles.